### DISCUSSION TITLE
Use hotkeys to remap keys? (`{`, `}`, `|`, `~` on a AZERTY keyboard)

### DISCUSSION THREAD

**Discussion by melMass** [deskflow/deskflow#7938] [Q&A]
Hi,

I'm trying to understand how to use hotkeys in the server configuration. I'm using a macbook as the server and windows and another mac as clients. 
The macos <-> macos keymapping works fine but on windows I can't find how to map it properly.
I've seen also an `alias` section (under each machine config) but couldn't understand if it's related or not.
I'm trying to remap those keys (all on an AZERTY keyboard):

| on the mac (server) | on the windows workstation (client) |
| - | - |
|`Alt+)` | `}`|
|`Alt+(` | `{`|
|`Alt+n` | `~`|
|`Alt+Shift+L` | `\|`|

*edit: to be clear the left column is the shortcuts to write what's on the right column by default on mac, not some custom stuff, to make it clearer here is a table of the key mapping*

| on the mac (server) | on the windows workstation (client) |
| - | - |
|`Alt+)` | `AltGr+Plus`|
|`Alt+(` | `AltGr+4`|
|`Alt+n` | `AltGr+2`|
|`Alt+Shift+L` | `AltGr+6`|

**Comment by nbolton** [comment]
Interesting question. Not something I have tried. To clarify, you want `Alt+)` on the Mac server to type `}` on Windows client?
